ui/apps/notifications/src/components/pages/customize-notification-page.tsx

```diff
@@ -218,7 +162,7 @@ const CustomizeNotificationPage: React.FC<CustomizeNotificationPageProps> = ({
     // @TODO: save preferences somewhere, now we just save a key in localstorage to mark as set
     try {
       if (window.localStorage) {
-        localStorage.setItem('notification-preference', JSON.stringify('1')); // @TODO: where to save settings?
+        localStorage.setItem(NOTIF_REF, JSON.stringify(allStates)); // @TODO: where to save settings?
```

Comment:
use the `settings` service from the sdk to set these for the current user

Reply:
the `service` name that you need to specify in the `set` call is the notification app name